be/src/common/config.h

```diff
@@ -864,6 +864,9 @@ CONF_mBool(lake_print_delete_log, "true");
 CONF_mBool(lake_compaction_check_txn_log_first, "false");
 // Used to ensure service availability in extreme situations by sacrificing a certain degree of correctness
 CONF_mBool(experimental_lake_ignore_lost_segment, "false");
+CONF_mInt64(experimental_lake_wait_per_put, "0");
```

Comment:
indicate that this configuration unit is `millisecond`

Reply:
Updated